third_party/proto/cosmos_proto/cosmos.proto

```diff
@@ -3,7 +3,7 @@ package cosmos_proto;
 
 import "google/protobuf/descriptor.proto";
 
-option go_package = "github.com/regen-network/cosmos-proto";
+option go_package = "github.com/cosmos/cosmos-proto";
 
 extend google.protobuf.MessageOptions {
     string interface_type = 93001;
```

Comment:
Without `interface_type`  in `MessageOptions` `make proto-gen` yields the following [error](https://play.golang.org/p/fA_H5F_1YGb) . Therefore, I have included `interface_type` field to fix this error and generate `*.pb.go` files successfully.

Reply:
We should remove `interface_type`. Try maybe removing `interface_type` from `scripts/protocgen.sh` too?

Reply:
Tried to make it, but the problem is still persists. See [comment](https://github.com/cosmos/cosmos-sdk/pull/9933#commitcomment-55408036)